README.md

```diff
@@ -1,57 +1,50 @@
 
 ```yaml
-- name: Report build info to Faros
-  id: emit-build-info
-  uses: faros-ai/faros-cicd-github-action@main
+- name: Report code build to Faros
+  id: send-ci-event
+  uses: faros-ai/faros-cicd-github-action@v2.0.0
   with:
     api-key: ${{ secrets.FAROS_API_KEY }}
-    api-url: ${{ env.FAROS_API_URL }}
-    model: build
-    build-pipeline-id: build-deploy-workflow # unique id to identify the workflow running the build
-    status: Success                          # possible values - Success, Failure, Cancelled otherwise defaults to Custom
-    started-at: 1594938057000
-    ended-at: 1594948069000
+    event: CI
+    artifact: Docker://my-org/my-repo/artifactId
+    run-status: ${{ job.status }}                     # possible values - Success, Failed, Canceled
+    run-started-at: 1594938057000
+    run-ended-at: 1594948069000
 ```
 
-> :clipboard: Note: Whilst the `build-pipeline-id` is optional, it is recommended to provide one to uniquely identify this workflow in Faros since multiple GitHub workflows can have the same name. If the `build-pipeline-id` is not provided it will be generated from the workflow name in the lowercase format `GITHUB_ORG/REPO/GITHUB_WORKFLOW_NAME`, e.g `my-org/my-repo/deployment`.
+### Report a code deployment (CD Event) To Faros
 
-### Report a Deployment Event To Faros CI/CD Deployment Model
-
-To report a build event to Faros specify `deploy` in the `model` parameter and the deployment details. To ensure the build is correctly linked to the build, provide the build model keys, i.e. `build-id`, `build-pipeline-id`, `build-org-id`, `build-source`.
+To report an artifact deployment to Faros specify `CD` in the `event` parameter and include the `CD` required fields.
 
 ```yaml
-- name: Report deployment info to Faros
-  id: emit-deployment-info
-  uses: faros-ai/faros-cicd-github-action@main
+- name: Report deployment to Faros
+  id: send-cd-event
+  uses: faros-ai/faros-cicd-github-action@v2.0.0
   with:
     api-key: ${{ secrets.FAROS_API_KEY }}
-    api-url: ${{ env.FAROS_API_URL }}
-    model: deploy
-    deploy-id: deploymentId
-    deploy-app-name: MyService               # name of the application being deployed
-    deploy-app-platform: ECS                 # platform application is deployed on
-    deploy-platform: CodeDeploy              # system used to orchestrate the deployment
-    build-id: build-id
-    build-pipeline-id: build-deploy-workflow
-    build-org-id: my-org
-    build-source: GitHub
-    started-at: 1594938057000
-    status: Queued                           # possible values - Canceled, Failed, Queued, Running, Success
+    event: CD
+    artifact: Docker://my-org/my-repo/artifactId
+    deploy: CodeDeploy://MyService/<env>/deploymentId # possible env values - Dev, Prod, Staging, QA
+    deploy-app-platform: ECS                          # the platform your application is deployed on
```

Comment:
can we not have this here? i'm trying to encourage people to not populate this since it needs to match platforms from incident management stuff, which as we know is quite messed up.

Reply:
it's a full example. in a simple one above it's not present

Reply:
Sure I can update that!

Reply:
oh, my bad. it's the CD event